layouts/partials/page-meta-links.html

```diff
@@ -0,0 +1,19 @@
+{{ if .Path }}
```

Comment:
Can we add whitespace to the lines in this file to make it easier to visually parse all the nested `if`s, or is that not possible / frowned upon?

Reply:
Thanks, I agree that it's hard to read. Yes we can add whitespace, but I think it will make it harder to compare against changes in the original file: (looks like it changed in the last 24 hours)

https://github.com/google/docsy/blob/master/layouts/partials/page-meta-links.html

Whenever we update Docsy, it's useful to update our local copies of these partial layout files to benefit from the latest changes.

WDYT?

Reply:
Sticking to the format of the upstream file makes sense. I believe there are ways in Git and Github to diff while ignoring whitespace changes but it's not really worth the added complexity IMO.

Reply:
Thanks, let's leave it as it is then.